sphinx/source/docs/dev_guide/server.rst

```diff
@@ -101,20 +101,20 @@ To work on the server, you'll need an understanding of Tornado's
 The Tornado documentation will be the best resource, but here are
 some quick things to know:
 
-- the Bokeh server is single-threaded, so it's important not to
+- The Bokeh server is single-threaded, so it's important not to
   write "blocking" code, meaning code that uses up the single
   thread while it waits for IO or performs a long computation. If
   you do this, you'll rapidly increase the latency seen by users
   of your application. For example, if you block for 100ms every
   time someone moves a slider, and ten users are doing this at
-  once, users could easily see 10*100ms=1s of lag... with only
+  once, users could easily see 10*100ms=1s of lag with only
   ten users.
-- in Tornado, nonblocking code is modeled with functions or
-  methods that return an instance of the ``Future`` class.  You
-  may have seen the ``@gen.coroutine`` decorator, which
+- In Tornado, non-blocking code is modeled with functions or
+  methods that return an instance of the ``Future`` class. You
+  may have seen the ``@gen.coroutine`` decorator. This decorator
```

Comment:
Just noting this whole section needs to be revisited for content at some point. Since moving to Python 3.6+ everything uses native coroutines and there is no more `gen.coroutine` etc. anywhere.

Reply:
Are those changes substantial enough to warrant adding a short note to this paragraph? How about adding this right after the sentence ending in line 116: 

> (Note: Bokeh now uses native coroutines introduced with Python 3.5. We hope to update the documentation soon to reflect these changes. Read more about native coroutines in the [Tornado documentation](https://www.tornadoweb.org/en/stable/guide/coroutines.html#native-vs-decorated-coroutines).)

Reply:
I would just say open a separate issue to keep track of the need. Anyone actually using this corner of the docs can probably navigate this change (or at least know enough to ask the right questions)

Reply:
Sounds good, I have opened issue #10341 to keep track of this!